### PR TITLE
Including output format option in tokei

### PR DESCRIPTION
**Pull request by mys721tx**
Includes output format option (CBOR, JSON, TOML, YAML) for tokei. The
features should be able to be broken down further.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by DomT4** [comment]
Is there any downside to just building these all in by default?

**Comment by mys721tx** [comment]
@DomT4 Apart from slightly longer build time, there is no downside I can think of. It probably will be more useful with all the features enabled.

**Comment by DomT4** [comment]
Let's build everything by default, without an opt-out. The formula has bottles so nobody with Homebrew based in `/usr/local` should even notice any longer build time.

**Comment by mys721tx** [comment]
@DomT4 Pushed. Awaiting for the CI check.

File: Formula/tokei.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Tokei < Formula
   depends_on "rust" => :build
 
   def install
-    system "cargo", "install", "--root", prefix
+    system "cargo", "install", "--root", prefix, "--feaures", "all"
```
Comment (DomT4) [inline Formula/tokei.rb]: Looks like there's a typo here, in `feaures`.

Comment (mys721tx) [reply Formula/tokei.rb]: Fixed in 27e5a9b.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/1017388fc062285847294bb1ca2636326434f2ba. Thank you for your contribution to Homebrew @mys721tx; we appreciate it! :smiley_cat:

**Pull request closed without merging**